src/store/index.js

```diff
@@ -243,6 +249,53 @@ const store = new Vuex.Store({
         loopIndex++;
       })
       commit(mutations.SET_ACCOUNT_CARDS, tokenDetailsArrayProcessed);
+    },
+    [actions.LOAD_DEEP_URL_CARD]: async function ({
+      commit,
+      dispatch,
+      state
+    }, {
+      tokenId
+    }) {
+      if (state.deepUrlCardNumber === null) {
+        commit(mutations.SET_DEEP_URL_CARD_NUMBER, tokenId)
+
+      } else {
+        if (state.contract) {
+          const contract = await state.contract.deployed();
+          let accountToken = await contract.tokenDetails(tokenId);
+          let gifter = accountToken[0]
+          let giftAmount = accountToken[1].toNumber()
+          let message = accountToken[2]
+          let extra = accountToken[3]
+          let cardIndex = accountToken[4]
+          let benefactorIndex = accountToken[5].toNumber()
+          if (state.cards) {
+            let cardInformation = state.cards.filter(card => {
+              return card.cardIndex === cardIndex.toNumber();
+            });
+            let accountCreatedCard = false
+            if (state.account) {
+              accountCreatedCard = (state.account.toLowerCase() === gifter.toLowerCase()) //if the current account created the card
```

Comment:
Comparing accounts works like this but we have found that if you checksum the accounts it a little less clunky and you can then set them in the store checksummed in the first place, you don't need to lowercase to equality check them `https://web3js.readthedocs.io/en/1.0/web3-utils.html#tochecksumaddress`

Reply:
cool. thanks for the recommendation. implemented this